clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1251,15 +1255,17 @@ class SyclKernelFieldChecker : public SyclKernelFieldHandler {
     assert(Util::isSyclAccessorType(Ty) &&
            "Should only be called on SYCL accessor types.");
 
-    const RecordDecl *RecD = Ty->getAsRecordDecl();
+    RecordDecl *RecD = Ty->getAsRecordDecl();
     if (const ClassTemplateSpecializationDecl *CTSD =
             dyn_cast<ClassTemplateSpecializationDecl>(RecD)) {
       const TemplateArgumentList &TAL = CTSD->getTemplateArgs();
+
```

Comment:
Extra lines here can also be removed. If checkAccessorType does not need to be modified in this patch, these changes will just be unrelated and random.